backend/db/setup.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Connect to the database and setup it with some default data.
+ */
+"use strict";
+
+const database = require('../db/database.js');
```

Comment:
Här ska variabeln _doc_ deklareras.

Reply:
Doc används inte vid reseten än då vi inte sätter in någon data än (filen kunde inte vara tom), därför är det bortkommenterat.